### ISSUE TITLE
Processes documentation

### ISSUE THREAD

**Issue by sapetnioc** [populse/capsul#270]
We must define how processes are documented. What is the language used to write the documentation and where it is written ? Processes are not isolated therefore there is a need for a documentation language allowing to define links between processes and toolboxes. Possible languages are markdown, RestructuredText and HTML. To choose we mus precise the targeted usage of the documentation (HTML pages generation, Sphinx doc generation, etc.).

**Comment by c-langlet** [comment]
Not aware of all of this, but Processes documentation should be as developer-friendly as possible, otherwise it may not be done.

**Comment by denisri** [comment]
Yes that's why a doc should be written only once, in the language allowing the needed customization.

Apparently there is a sphinx extension for markdown:
https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx
